javaslang/src/test/java/javaslang/AbstractValueTest.java

```diff
@@ -262,8 +276,8 @@ public void shouldConvertToHashMap() {
         //Check order - order is matter
         assertThat(map.mkString(", ")).isEqualTo(
                 "(110427681, 5), " +
-                "(663160547, 1), " +
-                "(1571254982, 9)");
+                        "(663160547, 1), " +
```

Comment:
you're splitting the string and reappending it?
Consider putting the operator in front to make it more obvious (and to format it better)